docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,65 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- The `gatsby-transformer-sharp` plugin installed
+- The `gatsby-plugin-sharp` plugin installed
+- A source plugin installed, such as `gatsby-source-filesystem`
+- An image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) in the `src` folder
+
+### Directions
+
+1. Make sure that the config file has the correct information.
+
+```JavaScript:title=gatsby-config.js
+
+ plugins: [
+   {
+     resolve: `gatsby-source-filesystem`,
+     options: {
+       name: `images`,
+       path: `${__dirname}/src/images`,
+     },
+   },
+   `gatsby-transformer-sharp`,
+   `gatsby-plugin-sharp`,
+ ],
+```
+
+Note: Note the gatsby src system path
+
+2.
+
+```query {
+     file(relativePath: { eq: "gatsby-astronaut.png" }) {
+       childImageSharp {
+         fluid(grayscale: true) {
+           ...GatsbyImageSharpFluid
+         }
+       }
+     }
+   }
+```
+
+Note: The relative path is the path that you config in gatsby src system.
+
+3. Next import image from "gatsby-image" line at the top
+
+ <Img fluid={data.file.childImageSharp.fluid}/>
```

Comment:
This step would also benefit from a complete React component example since users likely won't know where this code goes otherwise. There's another example on the `Using gatsby-image` guide, see Step 5 where they add the package and implement the `<Img>` component in the same file: https://www.gatsbyjs.org/docs/using-gatsby-image

Note, any code blocks should also be written in triple backticks -- having the docs site running locally can help you identify formatting.